cmd/kube-apiserver/app/server.go

```diff
@@ -556,7 +565,9 @@ func (s *APIServer) Run(_ []string) error {
 		Authorizer:                authorizer,
 		AdmissionControl:          admissionController,
 		DisableV1:                 disableV1,
-		EnableExp:                 enableExp,
+		EnableExtensions:          enableExtensions,
```

Comment:
Yes I can make this a generic struct:
```
ResourceOverrides struct {
  groupVersion string,
  resources util.Set,
  isEnableList bool
}
```
I was thinking of allowing overrides only for the extensions group and not v1 and that is why I did not put the generic struct, but will do it now.